paddle/fluid/framework/details/op_handle_base.h

```diff
@@ -31,7 +31,9 @@ constexpr char kLocalExecScopeName[] = "@LCOAL_SCOPE@";
 // It's responsible for populating necessary fields of ir::Node.
 class OpHandleBase {
  public:
-  explicit OpHandleBase(ir::Node *node) : node_(node) {}
+  explicit OpHandleBase(ir::Node *node) : node_(node) {
```

Comment:
There could add some Notes to tell the user that if the `OpHandleBase ` is created by `new`, user doesn't need call the `delete`.